draft-ietf-taps-impl.md

```diff
@@ -678,22 +681,22 @@ Connection Object:
 : TCP connections between two hosts map directly to Connection objects.
 
 Initiate:
-: Calling `Initiate` on a TCP Connection causes it to reserve a local port, and send a SYN to the Remote Endpoint.
+: CONNECT.TCP. Calling `Initiate` on a TCP Connection causes it to reserve a local port, and send a SYN to the Remote Endpoint.
 
 InitiateWithSend:
-: Early idempotent data is sent on a TCP Connection in the SYN, as TCP Fast Open data.
+: CONNECT.TCP with parameter "user message". Early idempotent data is sent on a TCP Connection in the SYN, as TCP Fast Open data.
 
 Ready:
-: A TCP Connection is ready once the three-way handshake is complete.
+: `Initiate` or `InitiateWithSend` returns without an error, i.e. the three-way handshake is complete.
```

Comment:
I don't quite follow this; Ready is the event that comes back after Initiate or InitiateWithSend to indicate that there was no error; the definition now is a bit tautological. The point of this was to say what the notion of being ready meant, which is just the part about the three-way handshake

Reply:
Agreed, I'll just remove this new bit again.